cobbler/actions/reposync.py

```diff
@@ -443,6 +443,9 @@ def rhn_sync(self, repo):
         if repo.arch != "":
             cmd = "%s -a %s" % (cmd, repo.arch)
 
+        if repo.arch == "":
+            cmd = "%s" % (cmd)
+
```

Comment:
We did not have enums but I am not sure if this is the correct. I would believe that this is more fitting for the following (which would mean we include the arch):

https://github.com/cobbler/cobbler/pull/2942/files#diff-77204e56ea1a781e3396e07cc7dea840201908389aa0a4bb0e00a6752d2a1e68R450-R451

Reply:
I'm still working through this component.  I am uncertain of the best approach.  According to `DNF-REPOSYNC(8)`:
```
       -a <architecture>, --arch=<architecture>
              Download only packages of given architectures (default is all architectures). Can be used multiple times.
```

Would simply adding `cmd = "%s -a %s -a noarch" % (cmd, repo.arch)` here be sufficient?

Reply:
I have inherited this code. I was not yet able to work my way through to this line. I am dependant on your call here. I don't have a test Cobbler at hand where I can experiment with such things sadly.

Reply:
In my local testing, this works for all arches.

Reply:
Then let's resolve this conversation and let it stay as-is.